src/main/resources/templates/practitioner/dashboard/error.html

```diff
@@ -6,8 +6,8 @@
 </head>
 <body>
 	<h1>ERROR: Patient ID does not exist, no valid condition entry or patient already existing.</h1>
```

Comment:
file name error.html could be more specific

Reply:
This is probably @JannesVolkens files, so I just moved them all from a "doctor" folder to "practitioner" folder